dev/archery/archery/bot.py

```diff
@@ -146,13 +141,20 @@ def handle_issue_comment(self, command, payload):
 
         comment = pull.get_issue_comment(payload['comment']['id'])
         try:
+            # Check privileges here to enable the bot to respond
+            # without relying on the handler.
+            allowed_roles = {'OWNER', 'MEMBER', 'CONTRIBUTOR'}
```

Comment:
I think the comment on lines 96-97 should stay with this line, as the linked docs explain the meaning of these roles.

Reply:
Thanks! Missed that.